src/spring/azext_spring/_params.py

```diff
@@ -861,6 +861,10 @@ def prepare_logs_argument(c):
                    help='Sensitive properties for environment variables. Once put, it will be encrypted and not returned.'
                         'Format "key[=value]" and separated by space.')
         c.argument('allowed_origins', arg_group='Cross-origin Resource Sharing (CORS)', help="Comma-separated list of allowed origins to make cross-site requests. The special value `*` allows all domains.")
+        c.argument('allowed_origin_patterns',
+                   arg_group='Cross-origin Resource Sharing (CORS)',
+                   options_list=['--allowed-origin-patterns', '--allow-origin-patterns'],
```

Comment:
We usually do not recommend using the passive voice for parameter names
```suggestion
                   options_list=['--allow-origin-patterns'],
```

Reply:
Since we want to keep the name align with API properties "allowedOriginPatterns", also we have another parameter "allowed_origins" already released.

Reply:
It is recommended to keep parameter naming style consistent with the CLI naming convention, as CLI customers perceive the consistency of naming of other similar CLI parameters more than the naming of REST parameters

Reply:
Additionally, it is recommended to deprecate and hide the existing parameter `--allowed-origins` and replace it with a new parameter `--allow-origins`

Reply:
I see.